woocommerce-services.php

```diff
@@ -820,8 +820,12 @@ public function rest_api_init() {
 		 * Delete this when the "v3" REST API is included in all the WC versions we support.
 		 */
 		public function wc_api_dev_init() {
-			if ( ! class_exists( 'WC_REST_Dev_Data_Continents_Controller' ) ) {
-				require_once( plugin_basename( 'classes/wc-api-dev/class-wc-rest-dev-data-controller.php' ) );
+			$rest_server = rest_get_server();
+			$existing_routes = $rest_server->get_routes();
+			if ( ! isset( $existing_routes['/wc/v3/data/continents'] ) ) {
+				if ( ! class_exists( 'WC_REST_Dev_Data_Continents_Controller' ) ) {
+					require_once( plugin_basename( 'classes/wc-api-dev/class-wc-rest-dev-data-controller.php' ) );
+				}
 				require_once( plugin_basename( 'classes/wc-api-dev/class-wc-rest-dev-data-continents-controller.php' ) );
```

Comment:
Shouldn't this be inside of the `class_exists()` check?

Reply:
It doesn't need to. If the `'/wc/v3/data/continents'` route doesn't exist, then neither `wc-api-dev` or the `v3` API exist.

In fact, the `class_exists` check isn't needed at all, I've just removed it in https://github.com/Automattic/woocommerce-services/pull/1448/commits/45a67a089627623fab8970aa8513b76af958e14a